### ISSUE TITLE
How to use training and validation sets together for training

### ISSUE THREAD

**Issue by qifeng22** [yifanzhang713/IA-SSD#55]

**Comment by qifeng22** [comment]
I want to use the training set and validation set together for training. Which code needs to be modified
我想将测试集和训练集代码一同用于训练，哪些代码需要改动
